source/persistence/MessageController.cpp

```diff
@@ -7,6 +8,11 @@
 #include <QDateTime>
 
 #include <QDebug>
+#include <QUrl>
+#include <QDir>
+#include <QFile>
+
+#include "CryptoHelper.h"
```

Comment:
Where is CryptoHelper used?

Reply:
This is used to retrieve the name of the attachment to remove